styles.css

```diff
@@ -589,3 +589,105 @@ ul.list-items.skill-items {
 .button-5 {
   padding-left: 10px;
 }
+
+form {
+  display: block;
+  margin: 0;
+}
+
+.form-class {
+  display: flex;
+  flex-direction: column;
+  margin-top: 12px;
+  align-items: center;
+}
+
+.form-btn {
+  color: #6070ff;
+  background-color: #fff;
+  border-radius: 8px;
+  padding: 12px 16px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border: 1px;
+  margin-top: 12px;
+  cursor: pointer;
+}
+
+.contact-info {
+  display: flex;
+  align-items: center;
```

Comment:
-  [ ] Please add padding-left to your input fields and textarea to match the figma design:
![image](https://user-images.githubusercontent.com/91486791/220391217-73a59fd6-2b04-43a1-8ee6-cb4bc476a27d.png)